release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,434 @@
+
+----
+openshift_use_crio=true
+----
+
+This setting pulls the *openshift3/cri-o* system container image from the
+link:https://access.redhat.com/containers[Red Hat Registry] by default. If you
+want to use an alternative CRI-O system container image from another registry,
+you can also override the default using the following variable:
+
+----
+openshift_crio_systemcontainer_image_override=<registry>/<repo>/<image>:<tag>
+----
+
+[NOTE]
+====
+The `atomic-openshift-node` service must be RPM- or system container-based when
+using CRI-O; it cannot be `docker` container-based. The installer protects again
+using CRI-O with `docker` container nodes and will halt installation if
+detected.
+====
+
+When CRI-O use is enabled, it is installed alongside `docker`, which currently
+is required to perform build and push operations to the reigstry. Over time,
+temporary `docker` builds can accumulate on nodes. You can optionally set the
+following to enable garbage collection, which adds a daemonset to clean out the
+builds:
+
+----
+openshift_crio_enable_docker_gc=true
+----
+
+When enabled, it will run garbage collection on all nodes by default. You can
+also limit the running of the daemonset on specific nodes by setting the
+following:
+
+----
+openshift_crio_docker_gc_node_selector={'runtime': 'cri-o'}
+----
+
+For example, the above would ensure it is only run on nodes with the `runtime:
+cri-o` label. This can be helpful if you are running CRI-O only on some nodes,
+and others are only running `docker`.
+
+See the link:http://cri-o.io/[upstream documentation] for more information on
+CRI-O.
+
+[[ocp-39-]]
+====
+
+
+[[ocp-39-]]
+====
+
+[[ocp-39-security]]
+=== Security
+
+[[ocp-39]]
+====
+
+[[ocp-39-storage]]
+=== Storage
+
+[[ocp-39-]]
+====
+
+[[ocp-39-scale]]
+=== Scale
+
+[[ocp-39-scale-cluster-limits]]
+==== Cluster Limits
+
+Updated guidance around
+xref:../scaling_performance/cluster_limits.adoc#scaling-performance-cluster-limits[Cluster
+Limits] for {product-title} 3.9 is now available.
+
+[[ocp-39-device-plugin]]
+==== Device Plug-in
+
+[[ocp-39-CPU-manager]]
+==== CPU Manager
+
+[[ocp-39-device-manager]]
+==== Device Manager
+
+[[ocp-39-hugepages]]
+==== HugePages
+
+
+
+
+[[ocp-39-networking]]
+=== Networking
+
+
+[[ocp-39-master]]
+=== Master
+
+[[ocp-39-]]
+====
+
+
+[[ocp-39-installation]]
+=== Installation
+
+
+
+[[ocp-39-diagnostics]]
+=== Diagnostics
+
+[[ocp-39-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-39-]]
+====
+
+[[ocp-39-developer-experience]]
+=== Developer Experience
+
+[[ocp-39-cli-plug-ins]]
+==== CLI Plug-ins
+
+CLI plug-ins are now fully supported.
+
+Usually called _plug-ins_ or _binary extensions_, this feature allows you to
+extend the default set of `oc` commands available and, therefore, allows you to
+perform new tasks.
+
+See xref:../cli_reference/extend_cli.adoc#cli-reference-extend-cli[Extending the
+CLI] for information on how to install and write extensions for the CLI.
```

Comment:
It might be nice to provide https://github.com/bostrt/openshift-sos-plugin as an example plugin! 
@bostrt @brenton Thoughts? Do we have time to make something like this possible?